mappings/net/minecraft/client/render/block/entity/BedBlockEntityRenderer.mapping

```diff
@@ -1,2 +1,7 @@
 CLASS net/minecraft/class_825 net/minecraft/client/render/block/entity/BedBlockEntityRenderer
 	FIELD field_4356 TEXTURES [Lnet/minecraft/class_2960;
+	METHOD method_3558 (Lnet/minecraft/class_4587;Lnet/minecraft/class_4588;ZLnet/minecraft/class_2350;Lnet/minecraft/class_2960;IZ)V
```

Comment:
Generic/bridge override?

Reply:
No, it's a private helper method. Haven't looked at what it does, but I mapped the parameters so that I could track more easily where parameters of some other method were coming from.